guides/gatherers.md

```diff
@@ -209,6 +219,15 @@ Example output (in Rhai):
 
 // node_0_ring0addr
 20;
+
+// node_list
+#{
+  "0": #{ "nodeid": 1, "ring0_addr": "10.80.1.11" },
```

Comment:
Maybe multi-line representation looks better, but well, this one is good as well

Reply:
Change to multiline 👍